### PR TITLE
separated animation data into two buffers and created toEnum function for resources

### PR DESCRIPTION
**Pull request by henrikvik**

### CODE REVIEW

File: Graphics/include/Renderer.cpp [hunk 1]
```diff
@@ -299,18 +301,18 @@ namespace Graphics
...
-        //    info.animationTimeStamp = time;
-        //    info.model = Resources::Models::AnimatedSummonUnit;
-        //    info.transform = SimpleMath::Matrix::CreateTranslation(0, 1, -3);
-        //    time += dt;
-        //    if (time > 5) time = 0;
-        //    return info;
-        //}(deltaTime));
+        QueueRender([](float dt) -> AnimatedRenderInfo
+        {
+            static float time = 0;
```
Comment (fkaa) [inline Graphics/include/Renderer.cpp:306]: static variabel i lambda? 🤢

File: Logic/source/Misc/HighScoreManager.cpp [hunk 2]
```diff
@@ -54,7 +54,7 @@ void HighScoreManager::loadFromFile()
     m_highScores.clear();
 
 	for (auto const& theScore : loadTo)
-        m_highScores.push_back({ theScore.ints.at(SCORE), theScore.strings.at(NAME) });
+//        m_highScores.push_back({ theScore.ints.at(SCORE), theScore.strings.at(NAME) });
```
Comment (fkaa) [inline Logic/source/Misc/HighScoreManager.cpp]: https://www.wired.com/2014/02/gotofail/

Visa detta till lukas 💯

**Review by fkaa — CHANGES_REQUESTED** [review]
Fixa bugs 🐛

**Review by henrikvik — COMMENTED** [review]
Ok!

**Review by fkaa — APPROVED** [review]

**Pull request merged**